extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson-common/runtime/src/main/java/io/quarkus/resteasy/reactive/jackson/common/runtime/ClientObjectMapper.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.resteasy.reactive.jackson.common.runtime;
```

Comment:
Actually, it's probably best if this is moved into the client module.
Also, we don't want to include `runtime` in the package.

Reply:
I have moved annotations to `io.quarkus.resteasy.reactive.jackson.common` package.  Annotations remain in `quarkus-resteasy-reactive-jackson-common` module.

Moving annotations to `quarkus-resteasy-reactive-jackson` module causes build to fail with following exception:

Deployment artifact io.quarkus:quarkus-rest-client-reactive-jackson-deployment::jar:999-SNAPSHOT is missing the following dependencies from its configuration: io.quarkus:quarkus-resteasy-reactive-jackson-deployment::jar, io.quarkus:quarkus-resteasy-reactive-deployment::jar, io.quarkus:quarkus-vertx-http-deployment::jar -> [Help 1]

Fixing that causes BasicTest in `integration-tests/rest-client-reactive` module to fail.  This seems way too complicated to fix.